src/Actions/Trackers/FocusTracker.cs

```diff
@@ -80,6 +80,26 @@ private void OnFocusChangedEventForSelectingElement(EventMessage message)
             }
         }
 
+        /// <summary>
+        /// microsoft/accessibility-insights-windows#1610: The "task switching"
+        /// interface that appears on Alt+Tab causes severe focus tracking
+        /// interference, especially on Sun Valley.
+        /// Therefore, this method detects whether this element is part of the
+        /// task switcher so that it can be filtered from focus tracking.
+        /// </summary>
+        /// <param name="element">The element to check.</param>
+        /// <returns>true if the element is part of the Alt+Tab task switching UI, false otherwise.</returns>
+        private bool IsTaskSwitcher(A11yElement element)
+        {
+            if (element == null) return false;
+            if (element.ProcessName != "explorer") return false;
+            return (
+                (element.ClassName == "Windows.UI.Input.InputSite.WindowClass" && element.ControlTypeId == ControlType.UIA_PaneControlTypeId) // Windows 11 "pane" window that sometimes appears
+                || (element.ClassName == "MultitaskingViewFrame" && element.ControlTypeId == ControlType.UIA_WindowControlTypeId) // Windows 10 task switcher
```

Comment:
Do we need special handling for Win10? We've only had complaints on Win11, so I'd be inclined to only address that

Reply:
I suppose not – it's kind of an optimization anyway.